integration_tests/e2eReferDisabled/refer.cy.ts

```diff
@@ -203,6 +203,20 @@ context('Refer', () => {
     notFoundPage.shouldContain404H2()
   })
 
+  it("Doesn't show the additional information page for a submitted referral", () => {
```

Comment:
again, would it be helpful to be more explicit that an error page is shown? (If I've understood correctly what's meant by `Page.verifyOnPage(NotFoundPage)`! )

Reply:
As discussed on the other PR, I'll leave this as-is for now as it's going to be removed very soon and is consistent with other tests.